solutions/usaco-1011.mdx

```diff
@@ -63,3 +63,39 @@ public class triangles{
 </JavaSection>
 
 </LanguageSection>
+
+<LanguageSection>
+ 
+<PySection>
```

Comment:
move this pysection into the languagesection above. you only need a new languagesection if it's a different method